DigitalLearningSolutions.Web.Tests/ViewModels/LearningMenu/IntitialMenuViewModelTests.cs

```diff
@@ -25,13 +26,21 @@ public void Initial_menu_should_have_name()
                 centreName,
                 bannerText
             );
+            const string sectionName = "TestName";
+            const bool hasLearning = true;
+            const double percentComplete = 12.00;
+            var section = new CourseSection(sectionName, hasLearning, percentComplete);
+            expectedCourseContent.Sections.Add(section);

```

Comment:
You're not testing the sections here so no need to add a section. Worth testing that the sections list gets converted to sections view model list in a separate test 🙂